### DISCUSSION TITLE
No access to assets from public?

### DISCUSSION THREAD

**Discussion by Beiri22** [directus/directus#6489] [Archive]
I cannot request an asset (image) without authentication through the link https://server/assets/img-id-here, I just get an Forbidden Error. Is that intended behaviour? I don't see any acces / permission roles for the assets in the roles&permission settings. It would be nice to get some help here...

Directus 9.0.0.rc80

**Comment by rijkvanzanten** [comment]
Assets share permissions with files 👍🏻 If you give read access to (certain) files, those assets can be downloaded as well

**Reply by NirvanKS** [reply]
I can't seem to find where I can give read access to files, where do I configure this? Is it through the API or the App? Also, should it be possible to allow Public to have access to the assets endpoint?

**Reply by rijkvanzanten** [reply]
You can go to Settings > Roles & Permissions > Public, and change the read permissions for the Files collection 👍🏻

**Reply by NirvanKS** [reply]
Thank you, it works
